ui_framework/components/menu/menu.js

```diff
@@ -0,0 +1,39 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const getClassName = ({ className, contained = false }) =>
+  classNames('kuiMenu', className, {
+    'kuiMenu--contained': contained
+  });
+
+
+const KuiMenu = ({
+  contained,
+  className,
+  children,
+  ...rest
+}) => {
+  return (
+    <ul
+      className={getClassName({
```

Comment:
I don't see any point in extracting a `getClassName`. I'd just inline this above the `return` here

```
const className = classNames('kuiMenu', className, {
  'kuiMenu--contained': contained
});
```